featuretests/cmd_juju_relation_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package featuretests
+
+import (
+	"bytes"
+	"os"
+
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/juju/osenv"
+	jujutesting "github.com/juju/juju/juju/testing"
+)
+
+type CmdRelationSuite struct {
+	jujutesting.JujuConnSuite
+	apps []string
+}
+
+func (s *CmdRelationSuite) SetUpTest(c *gc.C) {
+	s.JujuConnSuite.SetUpTest(c)
+	os.Setenv(osenv.JujuModelEnvKey, "")
+
+	s.apps = []string{"wordpress", "mysql"}
+	for _, app := range s.apps {
+		ch := s.AddTestingCharm(c, app)
+		s.AddTestingService(c, app, ch)
+	}
+}
+
+func run(c *gc.C, command, expectedError string, args ...string) {
```

Comment:
Good idea \o/ I'll move this to the package file and will return error so that individual tests can check for failures if expected.